src/main/java/org/team1540/robot2018/Tuning.java

```diff
@@ -1,4 +1,193 @@
+  public static double ejectSpeedB = -0.5;
+
+
+  // ELEVATOR
+  @Tunable("Elevator P")
+  public static double elevatorP = 2;
+
+  @Tunable("Elevator I")
+  public static double elevatorI = 0.0025;
+
+  @Tunable("Elevator D")
+  public static double elevatorD = 10;
+
+  @Tunable("Elevator F Upwards")
+  public static double elevatorFGoingUp = 2.5575;
+  @Tunable("Elevator F Downwards")
+  public static double elevatorFGoingDown = 0.75;
+  @Tunable("Elevator I-Zone")
+  public static int elevatorIZone = 100;
+
+  @Tunable("Elevator Tolerance")
+  public static double elevatorTolerance = 50;
+
+  @Tunable("Elevator Max Acceleration")
+  public static int elevatorMaxAccel = 300;
+
+  @Tunable("Elevator Cruise Velocity")
+  public static int elevatorCruiseVel = 400;
+
+  // @Tunable("Elevator Ground Position")
+  public static double elevatorGroundPosition = 5;
+
+  // @Tunable("Elevator Exchange Position")
+  public static double elevatorExchangePosition = 500;
+
+  // @Tunable("Elevator Front Switch Position")
+  public static double elevatorFrontSwitchPosition = 2900;
+
+  // @Tunable("Elevator Scale Lower Position")
+  public static double elevatorScalePosition = 7400;
+
+  // @Tunable("Elevator Low Scale Position")
+  public static double elevatorLowScalePosition = 6200;
+
+  // @Tunable("Elevator Obstacle Position")
+  public static double elevatorObstaclePosition = 1300;
+
+  // @Tunable("Elevator Obstacle Upper Position")
+  public static double elevatorObstacleUpperPosition = 3750;
+
+  @Tunable("Max Elevator Deviation")
+  public static double maxElevatorDeviation = 200;
+
+  // WRIST
+  @Tunable("Wrist P")
+  public static double wristP = 10.0;
+
+  @Tunable("Wrist I")
+  public static double wristI = 0;
+
+  @Tunable("Wrist D")
+  public static double wristD = 0;
+
+  @Tunable("Wrist F")
+  public static double wristF = 1.364;
+
+  @Tunable("Wrist I Zone")
+  public static int wristIzone = 0;
+
+  @Tunable("Motion Cruise Velocity")
+  public static int wristCruiseVelocity = 600;
+
+  @Tunable("WRist Max Acceleration")
+  public static int wristMaxAccel = 1000;
+  @Tunable("Wrist Peak Current Limit")
+  public static int wristCurrentLimit;
+  @Tunable("Wrist Peak Duration")
+  public static int wristPeakDuration;
+
+  @Tunable("Wrist Stop Tolerance")
+  public static double wristTolerance = 50;
+
+  // @Tunable("Wrist Out Position")
+  public static double wristOutPosition = 8250;
+  // @Tunable("Wrist Back Position")
+  public static double wristBackPosition = 0;
+  // @Tunable("Wrist Transit Position")
+  public static double wristTransitPosition = 3900;
+  // @Tunable("Wrist 45 Back Position")
+  public static double wrist45BackPosition = 1500;
+  // @Tunable("Wrist 45 Forward Position")
+  public static double wrist45FwdPosition = 6200;
+
+  @Tunable("Max wrist deviation")
+  public static double maxWristDeviation = 200;
+
+  // WINCH
+  @Tunable("Winch Out Speed")
+  public static double winchOutSpeed = 1; // 0.75
```

Comment:
rename to winch in (winch never goes out)

Reply:
This is actually never used, so I’m just deleting it.